modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -1,41 +1,47 @@
+
+The Mule app can also do nothing other than expose the tokenization service as a pass-through. 
+
+[NOTE]
+Because the tokenization service accepts only TLS traffic, you must configure TLS for the Mule app and enable the LastMile security. 
+
+== Prerequisites
 
 To configure the tokenization service, you need the following:
 
 * Runtime Fabric 1.1.153 or later with inbound traffic configured. See xref:1.2@runtime-fabric::index.adoc[the Runtime Fabric documentation].
 * A secret group to store the tokenization table encryption keys. The tokenization service creates and stores its own keys. See xref:asm-secret-group-concept.adoc[Secrets Manager documentation].
 * A tokenization format, which describes the format and how data is tokenized. See xref:tokenization-formats.adoc[Tokenization Formats].
-* A tokenization service that handles the actual tokenization requests and also tokenizes and detokenizes the data. For security purposes, an API Gateway must protect this service.
-* API Gateway, which proxies requests to the tokenization service, provides authorization and authentication, and provides JSON protection for the tokenization service.
-
-
-image::tokenization-setup-example-diagram.png[Tokenization Examples]
 
-In this diagram, a separate zone segregates the tokenization service from the general Runtime Fabric. This segregation is based on security needs.
+//image::tokenization-setup-example-diagram.png[Tokenization Examples]
 
-For example, if sensitive data is produced from a Mule application serving external traffic (Client -> Edge -> Mule application), this application might be included in the sensitive data zone.
+//In this diagram, a separate zone segregates the tokenization service from the general Runtime Fabric. This segregation is based on security needs.
 
-== Prerequisites
+//For example, if sensitive data is produced from a Mule app serving external traffic (Client -> Edge -> Mule application), this app might be included in the sensitive data zone.
 
-=== Runtime Fabric 1.1.153 or Later with Inbound Traffic
+////
+== Runtime Fabric 1.1.153 or Later with Inbound Traffic
 
 This example shows a Runtime Fabric named "rtf231", which has inbound traffic that is using a certificate and private key in the secret group “alphatoken”. If you already have a Runtime Fabric and secret group configured, use those names in places of the ones in the example.
 
 image::tokenization-example-rtf-ingress-config.png[Example Runtime Fabric with Inbound Traffic]
 
 For more information, see xref:1.2@runtime-fabric::enable-inbound-traffic.adoc[Runtime Fabric Inbound Traffic Configuration].
+////
+
+=== What to Know About Secret Groups and Tokenization
 
-=== Secret Group
+Each tokenization service you create must have a corresponding, dedicated secret group in Secrets Manager. No other tokenization service should use this secret group and no secrets outside of this tokenization service should be stored in that secret group.
```

Comment:
Replace all shoulds. Just tell the customer what to do, as in "Don't use this secret group with any other tokenization service..."